imports/plugins/core/system-info/server/no-meteor/schemas/schema.graphql

```diff
@@ -0,0 +1,21 @@
+extend type Query {
+  systemInformation(shopId: ID!): SystemInformation!
+}
+
+"Represents Reaction Plugin"
+type Plugin {
+  # name of plugin
+  name: String
+  # version of plugin
+  version: String
```

Comment:
`#` is a comment and we only use those sparingly for information that is useful in the file but need not be in the public docs. String literals are descriptions, which are used to build API docs, and we require those for everything. The descriptions should also start with a capital letter and be in sentence case. In this case:

```graphql
"Information about an API plugin"
type Plugin {
  "The plugin name"
  name: String!

  "The installed version of this plugin"
  version: String
}

"Information about the Reaction system"
type SystemInformation {
  "The core API version"
  apiVersion: String!

  # TODO: data version
  # TODO: mongo version

  "The installed API plugins"
  plugins: [Plugin]!
}
```

Note I also added `!` after the `String` type for `name` and `apiVersion`. This is because all plugins always have a name, and we want to inform API users that they can rely on that value being there without testing for `null`. I added one after `[Plugin]`, too, to indicate that we'll always return an array rather than `null`, even though it may be an empty array.

We also keep an empty line between each field as a matter of style.

You can add database version right now by doing this in the query function:

```js
const mongoInfo = await context.app.db.admin.serverStatus();
return {
  // ...
  mongoVersion: mongoInfo.version
};
```

There may be other useful info in `mongoInfo`, in which case you could make a separate `DatabaseInformation` type for it in the schema.

Reply:
i like the Idead of a databaseInformation Object as a child of the SystemInformation, i'll add that too